content/components/selectpanel.mdx

```diff
@@ -65,11 +47,7 @@ Select panels can be activated by either a regular [button](/components/button),
   src="https://github.com/primer/design/assets/980622/928a4bb2-c3f8-4620-9e37-42d9b91204c1"
 />
 
-If your selection doesn't trigger a series of events upon submission and doesn't include a checkbox in the footer, you may consider not adding a cancel/save button. In this scenario, when a selection is made, the select panel will instantly close similar to a menu. This behavior is indicated by a checkmark on the selected item.
-For example switching a branch is easily reversible and doesn't pose any risk of sending notifications to other users.
-
-However, if the select panel does cause a series of events upon selection or if the footer includes additional form elements like a checkbox, it becomes necessary to include cancel/save buttons. In such cases, the behavior is indicated visually by radio buttons, which allows users to confirm their choices, before submitting their selections.
-For example setting the wrong assignee on an pull request poses the risk of sending notifications to the wrong user therefore the cancel/save button is crucial.
+For single-item selection, if no consequential actions follow, the panel may close immediately, indicated by a checkmark on the selected item. However, when the selection triggers significant actions or the footer has additional elements (like a checkbox), declarative buttons are necessary. In such cases, the behavior is indicated visually by radio buttons, which allows users to confirm their choices, before submitting their selection.
```

Comment:
I'd like to encourage people towards actionmenu for _these_ instances. Unless the list is large enough to warrant a type-ahead filter, then we should use actionmenu. One that springs to mind is License chooser which has maybe 10 licenses, and can display the list all at once without requiring typeahead.

Reply:
That makes sense. Since this section is focusing on the options of the component, we could create a new section under "[Usage guidelines](https://primer-3de09d22a0-26441320.drafts.github.io/components/selectpanel#usage-guidelines)" to point that out, thoughts?

Reply:
Sounds like a good idea.

Reply:
More prominent and probably the best place in the [doc](https://primer-3de09d22a0-26441320.drafts.github.io/components/selectpanel):

![Screenshot showing the single select section with a info box that clarifies about action menu](https://github.com/primer/design/assets/3369400/b9031c69-ccb2-499d-997d-e7f62c8298b5)